source/appModules/audacity.py

```diff
@@ -1,15 +1,14 @@
 #appModules/audacity.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2010 NVDA Contributors <http://www.nvda-project.org/>
+#Copyright (C) 2006-2018 NVDA Contributors <https://www.nvaccess.org/>

```

Comment:
The updated copyright header should be:

# Copyright (C) 2006-2018 NV Access Limited, yourname

Reply:
Don't worry about hash sign.